### DISCUSSION TITLE
Generation of multiple files by tags

### DISCUSSION THREAD

**Discussion by jorge-plaza** [oapi-codegen/oapi-codegen#1715] [Q&A] [answered]
Hi,
I can't find any option that allows the generation of multiple output files based on their tags or other way of grouping. I think it would be useful to have that option, e.g. if you have a tag for `dog` and another for `cat` you get two `ServerInterface` with the methods for those tags. In this way you wouldn't need to have a huge implementation in case you have a huge list of endpoints.

**Comment by jamietanna** [comment, marked as answer]
https://github.com/oapi-codegen/oapi-codegen/issues/1513 is probably the feature request you're looking for?

**Reply by jorge-plaza** [reply]
I think that's it. I can try to contribute and open a PR
